packages/dds/test-dds-utils/src/ddsFuzzHarness.ts

```diff
@@ -420,29 +562,19 @@ function runTest<TChannelFactory extends IChannelFactory, TOperation extends Bas
 	itFn(`seed ${seed}`, async () => {
 		const random = makeRandom(seed);
 		const containerRuntimeFactory = new MockContainerRuntimeFactoryForReconnection();
-		const clients = Array.from({ length: options.numberOfClients }, (_, index) => {
-			const dataStoreRuntime = new MockFluidDataStoreRuntime();
-			const channel: ReturnType<typeof model.factory.create> = model.factory.create(
-				dataStoreRuntime,
-				// Use friendly IDs for a reasonable number of DDSes for easier debugging.
-				index < 26 ? String.fromCodePoint(index + 65) : random.uuid4(),
-			);
+		const clients = Array.from({ length: options.numberOfClients }, (_, index) =>
+			createClient(
+				containerRuntimeFactory,
+				model.factory,
+				makeFriendlyClientId(random, index),
+			),
+		);
 
-			const containerRuntime =
-				containerRuntimeFactory.createContainerRuntime(dataStoreRuntime);
-			const services: IChannelServices = {
-				deltaConnection: containerRuntime.createDeltaConnection(),
-				objectStorage: new MockStorage(),
-			};
-
-			channel.connect(services);
-			return { containerRuntime, channel };
-		});
+		const summarizerClient = createClient(containerRuntimeFactory, model.factory, "summarizer");
```

Comment:
should we create the summarizer first, then create all other clients from that using the summary technique? that seems to match our normal load flow, which is loading from the attachment summary